pkg/agent/openflow/pipeline.go

```diff
@@ -2215,6 +2215,7 @@ func (f *featureNetworkPolicy) dnsPacketInFlow(conjunctionID uint32) binding.Flo
 // proxyAll is disabled, currently there is no way to distinguish kubelet generated traffic from kube-proxy proxied traffic
 // only by matching the Antrea gateway IP. There is a defect that NodePort Service access by external clients will be
 // masqueraded as the Antrea gateway IP to bypass NetworkPolicies. See https://github.com/antrea-io/antrea/issues/280.
+// localProbeFlows also generates the flow to match the hairpin service packets and forward them to stageConntrack directly.
```

Comment:
I didn't mean moving it here, the new flow doesn't fall into this category and would look confusing. It should be where it was put in the beginning, just change the GoTo action.

Reply:
Got it. Changed.